torBot.py

```diff
@@ -130,6 +135,16 @@ def main(conn=False):
     if args.update:
         updater.updateTor()
         exit()
+    if args.database:
+        if 'DATABASE_URL' and 'DATABASE_NAME' and 'DATABASE_USERNAME' and 'DATABASE_PASSWORD' in os.environ:
```

Comment:
Turn this code into it's own function and see if this fixes the complexity issue

Reply:
done